app/code/Magento/Quote/Model/Quote/Address.php

```diff
@@ -139,6 +139,8 @@ class Address extends AbstractAddress implements
     const ADDRESS_TYPE_BILLING = 'billing';
 
     const ADDRESS_TYPE_SHIPPING = 'shipping';
+    
+    const CACHED_ITEMS_ALL = 'cached_items_all';
```

Comment:
Could you make this constant private in order to make your change backward compatible?
```suggestion
    private const CACHED_ITEMS_ALL = 'cached_items_all';
```